webhook.go

```diff
@@ -0,0 +1,67 @@
+package circleci
+
+import (
+	"context"
+	"fmt"
+)
+
+type Webhooks interface {
+	List(ctx context.Context, projectSlug string) (*WebhookList, error)
+	Create(ctx context.Context, webhook Webhook) (*Webhook, error)
+}
+type webhooks struct {
+	client *Client
+}
+
+type WebhookList struct {
+	Items         []*Webhook `json:"items"`
+	NextPageToken string     `json:"next_page_token"`
+}
+type Webhook struct {
+	URL           string   `json:"url"`
+	Name          string   `json:"name"`
+	SigningSecret string   `json:"signing-secret"`
+	Scope         Scope    `json:"scope"`
+	Events        []string `json:"events"`
+	VerifyTLS     bool     `json:"verify-tls"`
+}
+
+type Scope struct {
+	ID   string `json:"id"`
+	Type string `json:"type"`
+}
+
+func (w *webhooks) List(ctx context.Context, projectID string) (*WebhookList, error) {
+	if !validString(&projectID) {
+		return nil, ErrRequiredProjectID
+	}
+
+	u := fmt.Sprintf("webhook?scope-id=%s&scope-type=project", projectID)
```

Comment:
We use https://github.com/google/go-querystring
You don't have to specify the query strings here instead, create WebhookListOptions and add tags for corresponding ones.
https://github.com/grezar/go-circleci/blob/main/context.go#L43-L48